src/pages/test-coverage.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+
+import DefaultLayout from "../templates/DefaultLayout";
+import { Link } from "gatsby";
+
+export default function HowItWorks(data) {
+  return (
+    <DefaultLayout>
+      <section className="tc__hero type__copy">
+        <div className="limited-width">
+          <h1 className="type__headline__landing-hero">
+            Test coverage, solved.
+          </h1>
+          <h2 className="type__subline__landing-hero">
+            Saving your project time, money, and effort with the help of our
+            web3 smart <span className="no-break">contract experts</span>.
+          </h2>
+          <div className="tc__hero-image">
+            <img src="/images/tc-hero.svg" alt="Test Coverage Hero Image" />
+          </div>
+        </div>
+      </section>
+      <div className="type__copy">
+        <section className="tc__section limited-width tc__why">
+          <h2>
+            Projects are already choosing Code4rena{" "}
+            <span className="no-break">Test Coverage.</span>
+          </h2>
+          <p className="text">
+            Here’s why:{" "}
+            <a
+              href="https://medium.com/code4rena/new-to-code4rena-test-coverage-c548645404f9"
+              rel="noreferrer"
+              target="_blank"
+            >
+              {" "}
+              Code4rena Test Coverage
+            </a>{" "}
+            empowers projects to meet all of their priorities without
+            compromise, by providing projects with surge capacity for their
+            engineering teams pre-launch.
+          </p>
+          <div className="grid__one-by-two--break-m">
+            <div className="tc__saves-section">
+              <h3>Save time. </h3>
```

Comment:
Was this intentional to have the h3 exist outside the p tag but the other's are inside?